daphne_server/src/router/leader.rs

```diff
@@ -145,7 +145,9 @@ where
             daphne::DapResponse {
                 version: req.version,
                 media_type: DapMediaType::Collection,
-                payload: collect_resp.get_encoded_with_param(&req.version),
+                payload: collect_resp
+                    .get_encoded_with_param(&req.version)
+                    .expect("XXX"),
```

Comment:
@mendess how should we handle this encoding failure? Previously we would have panicked.

Reply:
Bump :)

Reply:
I had already answered here https://github.com/cloudflare/daphne/pull/473#discussion_r1452172468

Reply:
Yup, I saw! I resolved this comment but also should have commented. Sorry about that